android/player/BUILD

```diff
@@ -26,13 +26,26 @@ distribution(
     maven_coordinates = "com.intuit.playerui:android:%s" % VERSION,
 )
 
-kt_jvm_junit5_test(
+kt_player_test(
     name = "player-tests",
-    srcs = glob(["src/test/java/**"]),
-    associates = [":player-kotlin"],
-    kotlinc_opts = "//jvm:test_options",
-    test_package = "com.intuit.playerui.android",
-    deps = [":player"] + test_deps,
+    srcs = [
+        "src/test/java/com/intuit/playerui/android/AndroidPlayerTest.kt",
+        "src/test/java/com/intuit/playerui/android/AssetContextTest.kt",
+        "src/test/java/com/intuit/playerui/android/BrokenAssetTest.kt"
+    ],
+    associates = [":player-kotlin"]
+)
+
+filegroup(
+    name = "manifest",
+    srcs = ["src/main/AndroidManifest.xml"],
+    visibility = ["//visibility:public"]
+)
+
+kt_jvm_library(
```

Comment:
this is a terrible workaround for internals in `Constants.kt`. Since the vals are internal but they're used for the test utils in `utils/BUILD`, we make this target to use as a publicly visible associate

Reply:
in the latest commit this actually encapsulates everything, so then i can move `BrokenAssetTest.kt` file back to where it belongs and get rid of this test target above and place it into the test package. I just don't know if this is the way we want to go